### DISCUSSION TITLE
What is "NativeLink" for?

### DISCUSSION THREAD

**Discussion by awareness481** [vercel/next.js#16912] [Help]
I was poking through the [next-site](https://github.com/vercel/next-site) repo's code and saw this import statement;

```js
import NativeLink from 'next/link';
``` 

on [this file](https://github.com/vercel/next-site/blob/master/components/docs/text/link.js)

So I was wondering if anybody could tell me what it does.

**Comment by c0b41** [comment]
`NativeLink` just nextjs link component for  router.
